upload/catalog/model/extension/shipping/zasilkovna.php

```diff
@@ -437,4 +441,20 @@ public function sessionCleanup() {
 			unset($this->session->data[self::KEY_CARRIER_PICKUP_POINT]);
 		}
 	}
+
+	/**
+	 * @param int $orderId
+	 */
+	public function deleteIfNotPacketaShipping($orderId) {
```

Comment:
Co použít název jen deleteOrders () ? Podmínka pokud není packeta shipping není jediná podmínka, za které se to smaže.  Podmínky jsou čitelné z SQL dotazu.  Pokud bychom do názvu metody přidali všechny podmínky za kterých se to smaže , byl by název neúměrně dlouhý.

Reply:
jeste me napadlo deleteNonPacketaUnexported? deleteForeignUnexported?

Reply:
dokud nevymyslíme nic lepšího, dal bych:   deleteIfOrderNotPacketaShipping()